backend/src/gamestate/player.rs

```diff
@@ -7,6 +7,7 @@ pub struct Player {
     pub is_author: bool,
     pub cards: Vec<Card>,
     pub position: Option<usize>,
+    pub jwt: String,
```

Comment:
Jak jsem zmiňoval, při použití s JWT není potřeba ukládat token uživateli

Reply:
@mazanec202 Zmiňoval kde?

Reply:
> @mazanec202 Zmiňoval kde?

V [jednom z předchozích komentářů](https://github.com/mazanec202/rust-uno/pull/24#discussion_r789876657), dával jsi na něj srdíčko :D